plugins/de.cau.cs.kieler.klighd.lsp/src/de/cau/cs/kieler/klighd/lsp/KGraphDiagramGenerator.xtend

```diff
@@ -164,35 +166,30 @@ class KGraphDiagramGenerator implements IDiagramGenerator {
 	 * @param cancelIndicator Indicates, if the action requesting this translation has already been canceled.
 	 */
 	def SGraph toSGraph(KNode parentNode, String identifier, CancelIndicator cancelIndicator) {
-//        println("Starting SGraph generation!")
-//        val startTime = System.currentTimeMillis
         LOG.info("Generating diagram for input: '" + identifier + "'")
-	    
+
         kGraphToSModelElementMap = new HashMap
         textMapping = new HashMap
         modelLabels = new ArrayList
         images = new ArrayList
         idGen = new KGraphElementIDGenerator
         edgesToGenerate = new ArrayList
-        
+
         // generate an SGraph root element around the translation of the parent KNode.
         diagramRoot = new SKGraph => [
             type = 'graph'
             id = identifier
             children = new ArrayList
         ]
-        
+
         diagramRoot.children.addAll(createNodesAndPrepareEdges(#[parentNode], diagramRoot))
         // Do post processing.
         postProcess()
-        
-//        val endTime = System.currentTimeMillis
-//        println("SGraph generation finished after " + (endTime - startTime) + "ms.")
-        
+
         return if (cancelIndicator.canceled) 
-                   null 
-               else 
-                   diagramRoot
+               null 
```

Comment:
I suggest to always use the default java formatting in cases like this. Otherwise we are forced to not use the formatting feature at all or always limit it to specific areas. What is your opinion on that?